CONTRIBUTING.md

```diff
@@ -126,6 +126,22 @@ Before asking for a review it's best to wait for [Continuous Integration tests](
 Once a review has started both parties should attempt to respond to feedback in a timely manner.
 Do not approve the PR until you have either seen a successful CI test of the PR, or you can reasonably explain why a failure is unrelated to the code changes made in the PR (and documented in a PR comment).
 
+#### What can I expect in a code review?
+
+The review procedure is that the reviewer adds comments to the code if they have any issues with it. The person that raised the PR then responds by either:
```

Comment:
What about describe what we expect from 'comment' and 'request changes' and the differences between them?

Reply:
I will have a go at wording this comment aspect and incorporating request changes - thanks for the feedback.

Reply:
Does it look any better now from your point of view?

Reply:
Very clear and concise, thanks @tomjenkinson !